python/res/enkf/config_keys.py

```diff
@@ -197,10 +194,6 @@ class ConfigKeys:
     SHOW_HISTORY_KEY = _show_history_key
     LOG_FILE         = _log_file_key()
     LOG_LEVEL        = _log_level_key()
-    LSF_RESOURCE_KEY = _lsf_resources_key()
```

Comment:
Quite sure this hits everest: https://github.com/equinor/everest/blob/f7673d01edf65f8253064d483e1cf3ddcf8fff31/everest/detached/__init__.py#L296